### DISCUSSION TITLE
Franka Emika Model

### DISCUSSION THREAD

**Discussion by JeanElsner** [google-deepmind/mujoco#261] [Show and tell]
Hey everyone,

today I want to share with you a MuJoCo model of the Franka Emika Robot System (formerly known as Panda).
It's a torque-actuated model with dynamic parameters and textures specifically prepared for MuJoCo. While it's
ready to be used now and performs well, we plan to release an update based on our own system id soon.

You can find the model including a minimal working example [here](https://github.com/JeanElsner/panda_mujoco).

![Screenshot from 2022-04-28 14-18-30](https://user-images.githubusercontent.com/19858553/166812741-1a311e3c-73c6-4dcd-944d-045518b1a756.png)

**Comment by kevinzakka** [comment]
Thanks so much for sharing @JeanElsner !

**Comment by yuvaltassa** [comment]
Hi @JeanElsner,

You may have noted [this](https://mujoco.readthedocs.io/en/latest/models.html#arms), happy to collaborate and improve this further. We'll do some proper sysID soon, I hope 🙂
